### PR TITLE
WIP: Feature/co op modals

### PR DESCRIPTION
**Pull request by DiljotSG**
WIP pull request

- Redesigned the co-op page to be totally dynamic based on markdown files in /_co-op-companies/.
- Add a new company by creating a new markdown file in /_co-op-companies/.
- Add a new UMWICS member to a specific company by editing their respective markdown file.
- Company logos are found in /assets/img/logos/
- Member images are found in /assets/img/co-op/
- The modal layout is found in modal.html in the /co-op/ folder, and the co-op page layout is still in index.html

To-do:
- Need proper co-op summaries and statements for each member.
- Some members are missing photos.

### CODE REVIEW

File: join-us/index.html [hunk 1]
```diff
@@ -11,7 +11,12 @@
         </div>
     </div>
     <div class="row" style="margin-top: 15px; margin-bottom: 30px; text-align: center;">
-        <a href="https://wicsuofm.slack.com"><button class="ui primary button">Join Our Slack Here</button></a>
+        <a href="https://wicsuofm.slack.com">
```
Comment (stoykoj) [inline join-us/index.html]: Can you please change this page back to the original? I'll include it in another PR. Thanks!

File: _co-op-companies/A-bold.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: "Bold Commerce"
+identification: "bold"
+description: "Bold is an eCommerce company."
+link: "https://boldcommerce.com/"
+image: "assets/img/logos/bold-icon.png"
+width: "100px"
+members:
+  - name: "Elizabeth Kenyon"
```
Comment (stoykoj) [inline _co-op-companies/A-bold.md]: We'd like to just display first names.

File: join-us/index.html (hunk 1 above)
Comment (DiljotSG) [reply join-us/index.html]: Good eye Julia, I will fix that.

File: _co-op-companies/A-bold.md (hunk 2 above)
Comment (DiljotSG) [reply _co-op-companies/A-bold.md]: Currently, the name is just for identification in the markdown, and only the "summary" and "statement" Strings are pulled onto the page.

Comment (stoykoj) [reply _co-op-companies/A-bold.md]: Okay great! :+1:

File: join-us/index.html (hunk 1 above)
Comment (DiljotSG) [reply join-us/index.html]: I think it is good now. Let me know if you need anything else before merging.

**Comment by christopher-dG** [comment]
Is the format of the per-company files a Jekyll convention? Because if not, they are actually much closer to YAML format than Markdown, and they should probably have the `.yml` extension.

**Comment by DiljotSG** [comment]
@christopher-dG You are right. But here is why it works this way (I'm 90% sure), Jekyll is expecting to interpret a markdown file to build a web page, but I've turned that feature off in the config. Then we are just grabbing info from the collection (as defined in the config). The information for each item in the collection just happens to be in a YAML format (which is the same as the other HTML files in this repo, they all have a YAML header). As we aren't using the markdown to actually build the web page, the consequence of this is, the markdown portion is empty, and we only end up using the YAML header. So on the surface it appears misleading, but (I am pretty sure) this is fine.

[Collection information](https://jekyllrb.com/docs/collections/#accessing-collection-attributes)

**Comment by katharineking** [comment]
Hi Diljot, I think this layout looks great! A few things to consider: I would change the mouse select over each company's logo to make it clear that it goes to another page. Also, for exiting the member's list, I think it would be easier if the user could exit the pop up by clicking anywhere outside of the bubble instead of going to the right hand corned to exit the screen.

**Comment by DiljotSG** [comment]
What the page looks like now with semantic UI cards, probably broken on mobile.

screenshot 201

File: _co-op-companies/A-bold.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+description: "Bold is an eCommerce company."
+link: "https://boldcommerce.com/"
+image: "assets/img/logos/bold-icon.png"
+width: "100px"
+members:
+  - name: "Elizabeth Kenyon"
+    summary: "Elizabeth worked at Bold."
+    statement: "She had a great time."
+    image: "assets/img/co-op/elizabeth.jpg"
+  - name: "Julia Stoyko"
```
Comment (christopher-dG) [inline _co-op-companies/A-bold.md]: Image is Elizabeth, name is Julia. Mistake?

**Comment by christopher-dG** [comment]
Cool work.
Thoughts on using square logos for all companies? We could use [this](https://avatars2.githubusercontent.com/u/5262440?s=200&v=4) for Invenia and [this](https://media.glassdoor.com/sqll/34975/manitoba-hydro-electric-squarelogo-1460116685909.png) for Hydro (I couldn't find a bigger one).
Also should we maybe consider scaling the portrait images down a bit? They're pretty heavy and might make for slow page loads.

**Comment by DiljotSG** [comment]
Square logos? Yes.
Scale portraits down? Good idea.

**Comment by DiljotSG** [comment]
💯 
screenshot 206

File: co-op/index.html [hunk 4]
```diff
@@ -29,67 +32,37 @@
...
+    $(document).ready(function() {
+        {% for item in site.co-op-companies %}
+       $("#{{ item.identification }}-img").click(function() {
+           $("#{{ item.identification }}-modal").modal("show");
+       });
+       $("#{{ item.identification }}-close").click(function() {
+          $("#{{ item.identification }}-modal").modal("hide");
+       });
+       $('body').on('click', '.modal-backdrop.show', function(e) {  
+        $("#{{ item.identification }}-modal").modal("hide");
```
[the remark this answers is not in the document]
Comment (DiljotSG) [reply co-op/index.html]: That is a good question. Clicking on the modal does indeed close it.

[the remark this answers is not in the document]
Comment (DiljotSG) [reply co-op/index.html]: Correction: clicking on empty space in the modal does indeed close it.

[the remark this answers is not in the document]
Comment (DiljotSG) [reply co-op/index.html]: It actually works pretty intuitively. So, I'm going to merge this now 🎉 !

**Pull request merged**